### ISSUE TITLE
Confusing line at start of Windows updater

### ISSUE THREAD

**Issue by ryansturmer** [FabMo/FabMo-Edison-Image#4]
When the Windows batch file starts it says something like "Plug in board and reboot board"..don't understand what the reboot part is telling me to do?

**Comment by jlucidar** [comment]
yep, that's the message from the intel script, I think we can delete/modify it.
